core/src/core-api/featuretoggles.js

```diff
@@ -54,7 +46,37 @@ class LuigiFeatureToggles {
    * @example Luigi.featureToggles().getActiveFeatureToggleList();
    */
   getActiveFeatureToggleList() {
+    return [...get(this.featureToggleList)].filter(ft => !ft.startsWith('!'));
+  }
+
+  /**
+   * Get a list of active feature toggles
+   * @memberof FeatureToggles
+   * @return {Array} of feature toggles
+   * @example Luigi.featureToggles().getFeatureToggleList();
+   */
+  getFeatureToggleList() {
     return [...get(this.featureToggleList)];
   }
+
+  isValid(featureToggleName) {
+    if (isString(featureToggleName)) return true;
```

Comment:
**isString(featureToggleName)** is quite neat indeed, 
& I know that `featureToggleName && typeof featureToggleName === 'string'` looks a bit ugly, but it might be better than importing another package for one function. I guess here we can use the ugly version, as I presume importing a library might not be a luxury we can easily afford. Being that it can easily add to the bundle size in the long run might be better to avoid it as much as we can.

Reply:
Yeah, I thought this as well.
I will move it to `generic-helpers.js` and that can be reused in other places.